test/files/neg/macro-false-deprecation-warning/Impls_Macros_1.scala

```diff
@@ -1,5 +1,9 @@
+import scala.language.experimental.macros
 import scala.reflect.macros.blackbox.Context
 
+// don't emit deprecation warnings about identifiers like `macro` or `then`
+// when skimming through the source file trying to heal braces
+
```

Comment:
Example where it wasn't obvious what was being tested; also demonstrates that long test file names don't help much. In fact, long names are mostly obfuscating.